tests/emitters/gaussian_emitter_test.py

```diff
@@ -1,26 +1,41 @@
 """Tests for the GaussianEmitter."""
 import numpy as np
+import pytest
 
 from ribs.emitters import GaussianEmitter
 
 
 def test_properties_are_correct(archive_fixture):
     archive, x0 = archive_fixture
     sigma = 1
-    sigma0 = 2
-    emitter = GaussianEmitter(archive, x0, sigma, sigma0, batch_size=2)
+    batch_size = 2
+    emitter = GaussianEmitter(archive, x0, sigma, batch_size=batch_size)
 
-    assert (emitter.x0 == x0).all()
+    assert np.all(emitter.x0 == x0)
     assert emitter.sigma == sigma
-    assert emitter.sigma0 == sigma0
+    assert emitter.batch_size == batch_size
```

Comment:
why do you need to test batch-size here?

Reply:
For coverage, and also might as well.